src/main/java/eu/execom/pomodoroTeam/services/TeamService.java

```diff
@@ -0,0 +1,17 @@
+package eu.execom.pomodoroTeam.services;
+
+import eu.execom.pomodoroTeam.entities.TeamEntity;
+import eu.execom.pomodoroTeam.repositories.TeamRepository;
+import lombok.Data;
+import org.springframework.stereotype.Service;
+
+@Service
+
+
+
+
+public class TeamService {
```

Comment:
Please do not have floating annotations anywhere -> they should be right above the class, field or a method. Remove the empty spaces in between.